### DISCUSSION TITLE
Welcome to php-flasher.github.io Discussions!

### DISCUSSION THREAD

**Discussion by yoeunes** [php-notify/php-notify.github.io#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mohammedcha** [comment]
i just love it

**Reply by yoeunes** [reply]
Thank youuu @Mohammedcha (Casa 🇲🇦❤️), hope I can find time to document all of its parts.
